apps/root/src/pages/aggregator/swap-container/components/quote-selection/index.tsx

```diff
@@ -212,134 +91,95 @@ const QuoteSelection = ({
   const worseBy =
     selectedRoute && bestQuote && !isBestQuote && getWorseBy(bestQuote, selectedRoute, sorting, isBuyOrder);
 
-  const open = Boolean(anchorEl);
-  const id = open ? 'quotes-popover' : undefined;
+  let color: string | undefined = colors[mode].semantic.success.darker;
 
-  let color: string | undefined = colors[mode].semantic.success.primary;
-
-  if (!isBestQuote) {
-    color = colors[mode].semantic.error.primary;
+  if (!selectedRoute) {
+    color = colors[mode].typography.typo4;
+  } else if (!isBestQuote) {
+    color = colors[mode].semantic.error.darker;
   } else if (
     betterBy &&
     parseFloat(formatCurrencyAmount((isBestQuote ? betterBy : worseBy) || 0n, emptyTokenWithDecimals(18), 3, 2)) === 0
   ) {
     color = undefined;
   }
 
+  let diffLabel = formatSwapDiffLabel('0.00', sorting);
```

Comment:
no need to set a default value

Reply:
How would we set this 0.00 then? I didn't get this one